.devcontainer/devcontainer.json

```diff
@@ -3,10 +3,11 @@
     "build": {
         "dockerfile": "../Dockerfile",
         "context": "..",
-        "target": "visualizer",
+        "target": "dever",
```

Comment:
I think this would make more sense as a default as I imagine most people just want a docker image of Nav2 built from source

Reply:
That's fine. With foxglove now closed source, I'll have to get around to cleaning out the old visualizer stages anyhow.